docs/projects/cn0561/index.rst

```diff
@@ -0,0 +1,280 @@
+.. _cn0561:
+
+CN0561 HDL project
+===============================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`AD4134` is a quad channel, low noise, simultaneous sampling, precision 
+analog-to-digital converter (ADC), based on the continuous time sigma-delta 
+(CTSD) modulation scheme. This architecture inherently rejects signals around
+the ADC aliasing frequency band, giving the device its inherent antialiasing 
+capability, and removesthe need for a complex external antialiasing filter.
+
+This device has four independent converter channels in parallel, each with a CTSD 
+modulator and a digital decimation and filtering path. It enables simultaneous 
+sampling of four signal sources, with a maximum input bandwidth of 391.5 kHz.
+It supports a wide range of ODR frequencies, from 0.01 kSPS to 1496 kSPS wih less
+than 0.01 SPS adjustment resolution, allowing the user to granularly vary sampling
+speed to achieve coherent sampling.
+
+The :adi:`AD4134` supports two device configuration schemes: serial peripheral interface
```

Comment:
Please wrap the text to fit within 80 characters/line. You may use as a delimiter the --- symbols above.